code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -332,6 +332,26 @@
 	)
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
 
+/obj/item/clothing/suit/storage/serbian_overcoat
```

Comment:
Lacks most flags jackets and especially coats possess, such as temperature resistance, spawn tags, coverage and price